Research/Research/Research.h

```diff
@@ -2,7 +2,7 @@
 //  Research.h
 //  Research
 //
-//  Copyright © 2017 Sage Bionetworks. All rights reserved.
+//  Copyright © 2017-2019 Sage Bionetworks. All rights reserved.
```

Comment:
did anything else actually change in this file to justify updating the copyright? I mean I'm not sure but I would think just changing the copyright date would not be a copyrightable change....

Reply:
I think my reasoning was that the entire framework changed?